joomla/joomla-cms
# [5.1] Convert mod_title to service provider

### Summary of Changes

Converts the administrator title module to service provider.

### Testing Instructions

1. Make sure you have site module mod_title published
2. You should see module output in administrator area

![mod_title](https://github.com/joomla/joomla-cms/assets/59166858/1ad6f605-5a80-4b3d-9cd6-39245feb0da2)

### Actual result BEFORE applying this Pull Request

Module works as expected

### Expected result AFTER applying this Pull Request

Module works same way as before applying pull request

### Link to documentations
Please select:
